.github/workflows/update-manpage.yml

```
@@ -0,0 +1,76 @@
+name: Update manpage and completions
+
+on:
+  push:
+    paths:
+      - .github/workflows/update-manpage.yml
+    branches-ignore:
+      - master
+  pull_request:
+    types: ["opened", "synchronize", "reopened"]
+    paths:
+      - README.md
+      - Library/Homebrew/cmd/**
+      - Library/Homebrew/dev-cmd/**
+      - Library/Homebrew/completions/**
+      - Library/Homebrew/manpages/**
```

```suggestion
      - README.md
      - Library/Homebrew/cmd/**
      - Library/Homebrew/dev-cmd/**
      - Library/Homebrew/completions/**
      - Library/Homebrew/manpages/**
      - Library/Homebrew/cli/parser.rb
      - Library/Homebrew/env_config.rb
```

`cli/parser.rb` and `env_config.rb` contains the definitions for global args for formulae and cask commands and environment variables definitions, respectively. Changes to those could also cause manpage changes.

---

Based on your comment below, should we add the workflow file here, too?